packages/react-widgets/src/hooks/useWidgetFetch.js

```diff
@@ -10,7 +10,7 @@ export default function useWidgetFetch(
   { id, dataSource, params, global, onError }
 ) {
   // State
-  const [data, setData] = useState(null);
+  const [data, setData] = useState();
```

Comment:
Why this change here Sergio?

Reply:
https://github.com/CartoDB/carto-react/pull/370/files#diff-b6744b2ddd16d9987012b38f43d5800cd351e453fd471af46f6dbb649801e738R59

As you can see in that line, we define a default value. A default value can only be assigned in destructuring when it's undefined, not null.